packages/sls-mentor/src/rules/noTimeout/noTimeout.md

```diff
@@ -0,0 +1,31 @@
+---
+id: no-timeout
+title: no-timeout
+sidebar_label: no-timeout
+---
+
+# All Functions Have Timeout Configurations (no-timeout)
+
+Lambda Function timeout is configurable and should be configured for the use-case.
+Having it set at the maximum is rarely appropriate, unless processing a lot of data.
+The default is 3 seconds in AWS, but 6 with the Serverless Framework.
```

Comment:
If AWS has a default value for timeout, seems strange that it could be possible not to have a timeout configured on a specific function. Do you have a use case where this rule actually triggers an error ?

Reply:
Yes, this is a very good point; it does seem strange. I had picked up [this ticket](https://www.notion.so/m33/sls-mentor-047d75e5c4d94211a282ad5788351d9f?p=643016862d114fe78a15705fbf3caed5&pm=c) without thinking about the validity of the use case 😅